auto-run-dockerized-nondex/README.md

```diff
@@ -0,0 +1,74 @@
+# Dockerized Maven NonDex Runner
+
+This module provides a **generic and standardized Docker-based NonDex execution platform**, to detect flaky tests in Maven based Java projects.
+
+## Usage
+
+### Step 1: Build the Docker Image
+With the provided `Dockerfile` and `entrypoint.sh` files in your current directory, build the Docker image using:
+
+```bash
+docker build --build-arg JAVA_VERSION=17 -t nondex-runner .
+```
+### Step 2: Run the Docker Image
+Run the Docker container using the following command (sample below):
+
+```bash
+docker run \
+  -e REPO_URL=https://github.com/apache/myfaces-tobago.git \
```

Comment:
Perhaps worth including an example line on how to bind a local directory to the container's `/app` directory so that the student can debug / work with the repo and Nondex logs without having to work solely within the container.

Reply:
Sure. Good suggestion. I'll explain how local volume mounting to the Docker container works in my README.